config/database.yml

```diff
@@ -5,21 +5,29 @@
 #   gem 'sqlite3'
 #
 default: &default
-  adapter: sqlite3
+  <<: *default
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: localhost
+  port: 5432
   timeout: 5000
+  encoding: utf8
+  pool: 5
+  username: postgres
```

Comment:
This is hardcoded to use our local database, this won't work on prod because the credentials will be different so these should be environment variables.